packages/docs/community/release-process/smart-contracts.md

```diff
@@ -109,8 +138,12 @@ Deploying a new contract release should occur with the following process:
     <td>T+2w</td>
     <td>
       <ol>
-        <li>Tag the release on Github.</li>
-        <li>Run the smart contract release script to deploy the contracts to Baklava and submit a governance proposal.</li>
+        <li>Tag the release on Github using the finalized Release Notes.</li>
+        <li>On Tuesday: Run the <a href="#build-and-release-process">smart contract release script</a> in order to to deploy the contracts to Baklava as well as submit a governance proposal.
+         <ul>
+          <li> This involves sheperding the proposal through an expedited <a href="https://docs.celo.org/celo-codebase/protocol/governance"> governance process.</a></li>
```

Comment:
Why expedited?

Reply:
Yah this is a bit vague, and should probably be more detailed if we keep it included. I was under the impression that baklava governance process has much shorter stage timeframes, and that the devs pushing the upgrades are responsible for pushing the proposal through as opposed to the community at large.

Reply:
Ahh yeah. I got confused, because when I thought of `expedited` governance, I'm thinking of the hotfix route which of course is different for this one.

I know I'm contradicting myself here, but I would also be ok with omitting the shepharding of the proposal through Baklava governance, it could be pretty implied

Reply:
I'm OK with omitting it for now. I see how it's implied, but from my perspective I have no idea how this would be done (like, is the participation threshold really low on baklava in conjunction with quick timeframes? do we have an account we do this from?). As a new person this is a huge gray area

Reply:
Let's fill this gray area 

I would say participation is encouraged on Baklava, but I'm not sure we can expect a lot of participation. So it will be expedited in the sense that the timeframes are shorter, but also because there is less "friction" because less people will be participating, and few people/entities involved should easily make the proposal pass on Baklava